scripts/send-slack-notification-bot.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import json
+import base64
+import logging
+import os
+from subprocess import run, CalledProcessError
+
+def send_to_slack(webhook_url, release_info):
+    """Send release info to Slack channel"""
+    
+    message = f"""🚀 **Release {release_info['version']}**
+
+Links:
+- Jira: {release_info['jira_card_link']}
+
+Environment:
+- Cluster: {release_info['test_env']['cluster_name']}
+- NIC: {release_info['test_env']['nic']}
+- SECONDARY_NIC: {release_info['test_env']['secondary_nic']}
+- CNF Image: {release_info['test_env']['cnf_image_version']}
+- DPDK Image: {release_info['test_env']['dpdk_image_version']}
+"""
+
+    payload = {"text": message}
+    
+    try:
+        response = requests.post(webhook_url, json=payload)
+        if response.status_code == 200:
+            print("✅ Message sent to Slack!")
+            return True
+        else:
+            print(f"❌ Error: {response.status_code}")
+            return False
+    except Exception as e:
+        print(f"❌ Failed: {e}")
+        return False
+
+def read_file_content(file_path):
+    try:
+        with open(file_path, 'r') as f:
+            return f.read().strip()
+    except FileNotFoundError:
+        print(f"Warning: File not found at {file_path}. Returning empty string.")
+        return ""
+    except Exception as e:
+        print(f"Error reading file {file_path}: {e}")
+        return ""
+    
+def main():
+    webhook_url = os.environ.get("WEBHOOK_URL")
+    shared_dir = os.environ.get("SHARED_DIR")
+    
+    if not webhook_url or not shared_dir:
+        print("❌ Error: WEBHOOK_URL and SHARED_DIR environment variables must be set")
+        return
+    
+    version = read_file_content(os.path.join(shared_dir, "cluster_version"))
+    registry_url = "http://registry.stage.redhat.io/openshift4"
```

Comment:
Consider using HTTPS instead of HTTP for security:

```python
registry_url = "https://registry.stage.redhat.io/openshift4"
```

Also, consider making the registry URL configurable via environment variable for better flexibility:

```python
registry_url = os.environ.get("REGISTRY_URL", "https://registry.stage.redhat.io/openshift4")
```